src/models.py

```diff
@@ -236,6 +248,8 @@ def validation_step(self, batch: Tensor, batch_idx: int):
         # Calculate loss
```

Comment:
Perhaps make this comment also relate to perplexity? The comments can be better used here.

Reply:
Honestly, I think the code itself is so straightforward that the comments here aren't really necessary